docker/actinia-core/actinia.cfg

```diff
@@ -0,0 +1,44 @@
+[GRASS]
+grass_gis_start_script = /usr/local/bin/grass
+grass_database = /actinia_core/grassdb
+grass_user_database = /actinia_core/userdata
+grass_tmp_database = /actinia_core/workspace/temp_db
+grass_resource_dir = /actinia_core/resources
+grass_addon_path = /root/.grass7/addons/
+grass_gis_base = /usr/local/grass7
+grass_modules_xml_path = /usr/local/grass7/gui/wxpython/xml/module_items.xml
+grass_default_location = nc_spm_08
```

Comment:
I would set the grass_default_location to a location which is created in the start.sh.